spec/rubocop/cop/style/semicolon_spec.rb

```diff
@@ -105,6 +105,16 @@ def foo(a) x(1); y(2); z(3); end
       RUBY
   end
 
+  context 'with a multi-expression line without a semicolon' do
```

Comment:
Where's the multi-line expression here? :-)

Reply:
It’s not a multi-line expression, it’s a multi-expression line ... 😅

Reply:
Ops :-)